### PR TITLE
IHMC.CmapTools version 6.04

### PR DESCRIPTION
**Pull request by JoaquinPro64**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/53960)

### CODE REVIEW

File: manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: IHMC.CmapTools
+PackageVersion: 6.04
+PackageLocale: en-US
+Publisher: Institute for Human & Machine Cognition
+PackageName: CmapTools
```
Comment (ItzLevvie) [inline manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.locale.en-US.yaml:8]:
```suggestion
PackageName: IHMC CmapTools
```
This must match Control Panel.

File: manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: IHMC.CmapTools
+PackageVersion: 6.04
```
Comment (ItzLevvie) [inline manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.0.4.0
```

File: manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: IHMC.CmapTools
+PackageVersion: 6.04
```
Comment (ItzLevvie) [inline manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.yaml:5]:
```suggestion
PackageVersion: 6.0.4.0
```

File: manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: IHMC.CmapTools
+PackageVersion: 6.04
```
Comment (ItzLevvie) [inline manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.installer.yaml:5]:
```suggestion
PackageVersion: 6.0.4.0
```

File: manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: IHMC.CmapTools
+PackageVersion: 6.04
+PackageLocale: en-US
+Publisher: Institute for Human & Machine Cognition
+PackageName: CmapTools
+License: 2019
```
Comment (ItzLevvie) [inline manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will also need to update the version folder from `6.04` to `6.0.4.0`

**Comment by JoaquinPro64** [comment]
> You will also need to update the version folder from `6.04` to `6.0.4.0`

ok, just updated it

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> ok, just updated it

You will need to commit the changes using Visual Studio Online so the branch gets updated with your new changes.

**Comment by JoaquinPro64** [comment]
> cc @JoaquinPro64: I made a PR on your fork: [JoaquinPro64#2](https://github.com/JoaquinPro64/winget-pkgs/pull/2)

Just merged it

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**